### PR TITLE
Mirror: Cleaner Grenade

### PR DESCRIPTION
**Pull request by SimpleStation14**
###### `55ad9c7e433ebd0f58229157d2d7fd766b268ee4`

PR opened by <a href="https://github.com/DenisShvalov"> DenisShvalov</a> at 2024-02-21 14:47:43 UTC

---

PR changed 9 files with 63 additions and 3 deletions.

The PR had the following labels:
- No C#
- Changes: Sprites
- Status: Needs Review

---

<details open="true"><summary><h1>Original Body</h1></summary>

> Added Cleaner Grenade
> 
> ## About the PR
> Added Cleaner Grenades that creates foam of space cleaner of activate
> - Added 2 grenades in janitor belt on spawn
> - Added 2 grenades in custodial closet
> - Added 2 grenades in janitorial supplies crate
> 
> ## Why / Balance
> Big puddles of space lube 💀 
> 
> ## Media
> 
> https://github.com/space-wizards/space-station-14/assets/115770678/99fe98e1-c8fa-49d4-8658-a73519887606
> 
> - [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
> 
> :cl:
> - add: Added Cleaner Grenades that will help janitors in their work
> 
> 

</details>

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
This one's good, I'll set it to automerge for when another maintainer checks it because this is needed for the DV update queue.

**Review by FoxxoTrystan — COMMENTED** [review]
Aproved.

Grenade works, belt works, box are here, ect...
All seem good from my side.

image

File: Resources/Prototypes/Catalog/Fills/Boxes/service.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Catalog/Fills/Boxes/service.yml:14]:
```suggestion
      - state: flashbang

```

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml
Comment (DangerRevolution) [inline Resources/Prototypes/Entities/Clothing/Belt/belts.yml:226]: What does this do?

Comment (DangerRevolution) [reply Resources/Prototypes/Entities/Clothing/Belt/belts.yml:226]: Oh component whitelist.

**Review by DangerRevolution — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Boxes/service.yml
Comment (DangerRevolution) [reply Resources/Prototypes/Catalog/Fills/Boxes/service.yml:14]: can be merged after this

File: Resources/Prototypes/Entities/Clothing/Belt/belts.yml
Comment (FoxxoTrystan) [reply Resources/Prototypes/Entities/Clothing/Belt/belts.yml:226]: Yep

File: Resources/Prototypes/Catalog/Fills/Boxes/service.yml
Comment (FoxxoTrystan) [reply Resources/Prototypes/Catalog/Fills/Boxes/service.yml:14]: This is here for the Box Icon
image

Comment (DangerRevolution) [reply Resources/Prototypes/Catalog/Fills/Boxes/service.yml:14]: yuh but i like having a line at the end of the file, psychological thing and Death instilled it into me

**Review by DangerRevolution — APPROVED** [review]
fixed, nice PR

**Pull request merged**
